conda_recipe_v2_schema/model.py

```diff
@@ -316,6 +316,11 @@ class Python(StrictBaseModel):
         description="The path to the site-packages folder. This is advertised by Python to install noarch packages in the correct location. Only valid for a Python package.",
     )
 
+    version_independent: bool | JinjaExpr = Field(
+        default=False,
+        description="Whether the package is version independent or not. This is useful for 'abi3' packages that are OS specific, but not Python version specific.",
```

Comment:
maybe we could add a small hint for what is the jinja expresion? something along the lines
```
Whether the package is version independent or not or what is the pattern for this
```